### DISCUSSION TITLE
ScriptReference/PrefabUtility-prefabInstanceUpdated

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#657] [Notes]
# ScriptReference/PrefabUtility-prefabInstanceUpdated

https://docs.unity3d.com/ScriptReference/PrefabUtility-prefabInstanceUpdated.html

**Comment by andreasIsCoding** [comment]
Note that as of 2020.3, this delegate is called for EVERY change to the prefab instance objects, not just changes that propagate from a modified prefab.  This included changes made via the inspector, or changes made via scripts.
